### ISSUE TITLE
courses

### ISSUE THREAD

**Issue by rootsmusic** [JoseDeFreitas/awesome-youtubers#85]
I suggest separately categorizing courses, which are a sequenced playlist of videos about a subject.  I don't consider YouTuber creators, who upload random topics related to a subject, as "teachers".  Teachers are those who plan out an entire course.  Thanks.

**Comment by JoseDeFreitas** [comment]
Hello, @rootsmusic, thank you for opening this issue. As the name of the list implies, it is a list of awesome YouTubers that teach about technology, not a list of awesome courses about technology. I consider a teacher someone who explains a specific topic in a good way (in a way that students can understand easily), but not necessarily structured in a course. There is a list that enumerates awesome courses. If an awesome YouTuber uploads random topics (and by “random topics” I guess you mean a channel that isn't specifically pointed to just one topic), it should be added to this list.

In conclusion: I don't find any reason why the list should be separated; it enumerates awesome YouTubers that teach about technology, even if they do videos that are not inside a course (they must, of course, be awesome YouTubers, that is, YouTubers that know what they are talking about, have a good setup and can be understood by everyone).

If I misread what you wrote or you disagree with me, please send me a response so we can get to a synthesis.

**Comment by rootsmusic** [comment]
Ok @JoseDeFreitas, you're right that this list isn't titled "Awesome YouTube Teachers".  By the way, where can I find the list that enumerates awesome courses (I assume you mean on YouTube)?  Thanks.

**Comment by JoseDeFreitas** [comment]
@rootsmusic, I didn't understand you fully; I notice that you're looking for professional technology teachers. I wasn't so strict while saying the word “teachers”, as I consider a teacher (in this sense) a YouTuber that knows how to explain well (among other aspects), even if the YouTuber is not a professional teacher (academically speaking). Sorry for the misunderstanding.

I can't remember the exact list of awesome courses about technology (I visited it long ago), but with [a quick search](https://github.com/search?q=awesome+courses&type=) I found some good repositories (specifically the list from [prakhar1989](https://github.com/prakhar1989), where you can find good courses from well-known universities (taught by professional teachers). Hope it helps!
